### DISCUSSION TITLE
???

### DISCUSSION THREAD

**Discussion by zombiegrl** [Dpeta/pesterchum-alt-servers#74] [Q&A]
downloaded this for a friend, it refuses to open. i keep getting the same "unknown author" so it won't let me run it. i went through my firewall and allowed the app access but it still won't run?? if someone can tell me wtf to do.... yah idk some help would be lovely!

**Comment by Dpeta** [comment]
Being able to run the executable shouldn't have anything to do with a firewall, are you on Windows or macOS? :?c
- On Windows you gotta:
![Screenshot_20220330_183734](https://user-images.githubusercontent.com/69427753/160886668-9a3fcbde-c736-4f6d-a2bd-ed632ef0277d.png)
- & on macOS you can usually run it by right clicking & then pressing open from the drop-down menu explicitly. (Or you can do it like [this](https://www.howtogeek.com/205393/gatekeeper-101-why-your-mac-only-allows-apple-approved-software-by-default) or [this](https://support.apple.com/guide/mac-help/open-a-mac-app-from-an-unidentified-developer-mh40616/10.15/mac/10.15).)
